### DISCUSSION TITLE
Mr. Vlad, please make the SDXL better in Vlad diffusion, at least on the level of configUI.

### DISCUSSION THREAD

**Discussion by weirdlighthouse** [vladmandic/sdnext#1922] [General]
I just recently tried configUI, and it can produce similar results with less VRAM consumption in less time. but the node system is so horrible and confusing that it is not worth the time.

It lacks the Simplicity that kills the configUI for me.

**Comment by vladmandic** [comment]
well tuned sdnext is on-par or faster than comfy. the issue is what defaults to use?
for example, if i tune for nvidia rtx4090, its going to run awful on nvidia gtx2060.
and i'm frequently changing defaults, mostly according to community feedback.

**Reply by weirdlighthouse** [reply]
how much Ram is enough?

**Reply by weirdlighthouse** [reply]
seriously how much Vram I need, I am planning to upgrade, also Amd or NVidia

**Reply by SanDiegoDude** [reply]
Maybe you could perhaps add some presets for best quality, best performance, low system RAM, potato PC mode? I bet discord could come up with some good settings for each, then just put a big fat warning on it that YMMV - but would help with situations like what OP is complaining about - I had to get my settings tuned for it to run properly for me (and SDXL runs great now, though I must admit I'm using A1111 HRF currently, since I'm using my own custom model now that doesn't play nice with the refiner, pretty please with sugar on-top add compat for HRF and SDXL in safetensors mode like A1111 does)

**Reply by vladmandic** [reply]
Hires is definitely coming, it's a high priority item.
Just things are slightly delayed right now due to my travels.

**Reply by spreck** [reply]
@vladmandic why not have user contributed tunes to specific GPU's, selectable by a dropdown menu

**Comment by usigg3** [comment]
Any tips on how to optimize for a 2060?

**Comment by Sunsphire** [comment]
Optimization tips for different gpus would be great, especially for those like me that struggle with older hardware (2080). I'm not sure how that can work but it would be nice if the community can be involved in this.

**Reply by vladmandic** [reply]
Good starting point is to engage with community on discord and compile best settings for all different HW - I'd definitely be more than willing to publish it on wiki. And if possible, even add some optimizations for settings directly in code. But I cannot test every possibility myself - thats why community engagement is critical for any major open source project.
